test/Microsoft.Data.Sqlite.Tests/SqliteTransactionTest.cs

```diff
@@ -216,6 +217,22 @@ public void Dispose_works()
         }
 
         [Fact]
+        public void Serializable_locks()
+        {
+            using (var connectionA = new SqliteConnection("Data Source=testdb;Mode=Memory;Cache=Shared"))
+            using (var connectionB = new SqliteConnection("Data Source=testdb;Mode=Memory;Cache=Shared"))
```

Comment:
Yes, join the dark side... :smiling_imp: I prefer to stack using statements too, but it does go against our coding guidelines.